src/users/users.controller.spec.ts

```diff
@@ -780,4 +780,51 @@ describe('UsersController', () => {
       });
     });
   });
+
+  describe('POST /users/:id/token', () => {
+    describe('without api token', () => {
+      it('returns a 401', async () => {
+        const { body } = await request(app.getHttpServer())
+          .post('/users/0/token')
+          .expect(HttpStatus.UNAUTHORIZED);
+
+        expect(body).toMatchSnapshot();
+      });
+    });
+
+    describe('with invalid user id', () => {
+      it('returns a 404', async () => {
+        const user = await usersService.create({
+          email: faker.internet.email(),
+          graffiti: uuid(),
+          countryCode: faker.address.countryCode('alpha-3'),
+        });
+
+        const { body } = await request(app.getHttpServer())
+          .post(`/users/${user.id + 1}/token`)
+          .set('Authorization', 'Bearer test')
+          .expect(HttpStatus.NOT_FOUND);
+
+        expect(body).toMatchSnapshot();
+      });
+    });
+
+    describe('with a valid payload', () => {
+      it('create auth token', async () => {
+        const user = await usersService.create({
+          email: faker.internet.email(),
+          graffiti: uuid(),
+          countryCode: faker.address.countryCode('alpha-3'),
+        });
+
+        const { body } = await request(app.getHttpServer())
+          .post(`/users/${user.id}/token`)
+          .set('Authorization', 'Bearer test')
+          .expect(HttpStatus.CREATED);
+
+        expect(body.email).toEqual(user.email);
+        expect(body.url).toBeDefined();
```

Comment:
This is always defined given the response schema. Can we check that the substring `https://api.ironfish.network/login` is part of this `url`?